imap/caldav_alarm.c

```diff
@@ -396,6 +412,10 @@ static int send_alarm(struct get_alarm_rock *rock,
     }
     FILL_ARRAY_PARAM(event, EVENT_CALENDAR_ALARM_RECIPIENTS, recipients);
 
+    prop = icalcomponent_get_first_property(alarm, ICAL_UID_PROPERTY);
+    FILL_STRING_PARAM(event, EVENT_CALENDAR_ALERTID,
```

Comment:
Do we have to fetch the UID property again?  We already have the UID (if exists) in eid.ical_uid

Reply:
Does the UID of a VALARM need to match the embedding VEVENT UID? I don't think so, as it would mean we can't have multiple unique UIDs of VALARMs in an event.

Reply:
My mistake, I didn't realize that they were two different UIDs.  Yes, VALARM UID is different than VEVENT UID.